civitai-downloader-v2/src/api/streaming_search.py

```
@@ -0,0 +1,379 @@
+        Get current memory usage of the process.
+        
+        Returns:
+            Current memory usage in bytes
+        """
+        return self._get_current_memory()
+    
+    def _get_current_memory(self) -> int:
+        """Get current memory usage in bytes."""
+        try:
+            return self._process.memory_info().rss
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)